client/src/pages/Landing/Landing.js

```diff
@@ -20,8 +23,10 @@ export default function Landing() {
                             <h1 className="display-4"><em>My</em> EHR</h1>
                             <p className="lead">All the medical records you need in one, simple place.</p>
                             <hr className="my-4" />
-                       
-                            <Link to="/signup" className="btn btn-primary btn-lg">Get Started</Link>
+                       {(user === '' ?
```

Comment:
Change this condition to : (!user || user === '')

When the user first arrives with nothing in local storage, user is undifined.  And clicking 'get started' does nothing.  this change should fix it.